File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        int l=nums.size();
        int count=0;
        for(int i=0;i<l-1;i++){
            if(nums[i]>nums[i+1]){
                count++;
            }
        }
        if(nums[l-1]>nums[0]){
            count++;
        }
        if(count<=1){
            return true;
        }
        else{
            return false;
        }
        return true;
    }
};

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int cnt = 0;
        for(int i=0;i<nums.length;i++){
            int next = (i+1)%nums.length;
            if(nums[i]>nums[next]){
                cnt++;
            }
        }

        if(cnt>1){
            return false;
        }
        return true;
    }
}